### DISCUSSION TITLE
blog/2023-06-02-k3s-coredns-issue

### DISCUSSION THREAD

**Discussion by giscus** [crazychenz/vinnie.work#35] [vinnie.work Comments]
# blog/2023-06-02-k3s-coredns-issue

Overview

https://www.vinnie.work/blog/2023-06-02-k3s-coredns-issue

**Comment by x45dev** [comment]
This is a great investigation and very useful information. Thank you! 
I particularly liked the fact that you included details about investigating the SQLite db. I didnt come here looking for that, but it's great to now know how!

**Reply by crazychenz** [reply]
Thanks for reading!
